athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneEnvironmentProperties.java

```diff
@@ -0,0 +1,47 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation SDK
+ * %%
+ * Copyright (C) 2019 - 2024 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.neptune;
+
+import com.amazonaws.athena.connector.lambda.EnvironmentProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class NeptuneEnvironmentProperties extends EnvironmentProperties
+{
+    private static final String NEPTUNE_ENDPOINT = "neptune_endpoint";
+    private static final String NEPTUNE_PORT = "neptune_port";
+    private static final String NEPTUNE_CLUSTER_RES_ID = "neptune_cluster_res_id";
+    private static final String CLUSTER_RES_ID = "CLUSTER_RESOURCE_ID";
+    private static final String NEPTUNE_GRAPHTYPE = "neptune_graphtype";
+    private static final String GRAPH_TYPE = "GRAPH_TYPE";
+
+    @Override
+    public Map<String, String> connectionPropertiesToEnvironment(Map<String, String> connectionProperties)
+    {
+        Map<String, String> environment = new HashMap<>();
+
+        environment.put(NEPTUNE_ENDPOINT, connectionProperties.get("HOST"));
```

Comment:
we should consider putting the unified connector parameters into a single utility class

Reply:
sounds good. In athena-federation-sdk?